pkg/model/flow_content.go

```diff
@@ -98,6 +94,26 @@ func (p *BpfFlowContent) AccumulateAdditional(other *ebpf.BpfAdditionalMetrics)
 	if !AllZeroIP(IP(other.TranslatedFlow.Saddr)) && !AllZeroIP(IP(other.TranslatedFlow.Daddr)) {
 		p.AdditionalMetrics.TranslatedFlow = other.TranslatedFlow
 	}
+	// Accumulate interfaces + directions
+	accumulateInterfaces(&p.AdditionalMetrics.NbObservedIntf, &p.AdditionalMetrics.ObservedIntf, other.NbObservedIntf, other.ObservedIntf)
+}
+
+func accumulateInterfaces(dstSize *uint8, dstIntf *[MaxObservedInterfaces]ebpf.BpfObservedIntfT, srcSize uint8, srcIntf [MaxObservedInterfaces]ebpf.BpfObservedIntfT) {
```

Comment:
can u add UT for this make sure it handles all variances

Reply:
it's actually already covered but in another package, I'll move the test to the appropriate package

Reply:
test moved: https://github.com/netobserv/netobserv-ebpf-agent/pull/470/commits/52341d440775e6b31aa00d7bc751b7c44c360490